libfreerdp/utils/smartcard_call.c

```diff
@@ -1548,7 +1548,7 @@ static LONG smartcard_LocateCardsByATRA_Call(scard_call_context* smartcard, wStr
 }
 
 LONG smartcard_irp_device_control_call(scard_call_context* smartcard, wStream* out,
-                                       UINT32* pIoStatus, SMARTCARD_OPERATION* operation)
+                                       NTSTATUS* pIoStatus, SMARTCARD_OPERATION* operation)
```

Comment:
Isn't NTSTATUS a signed type (https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-dtyp/c8b512d5-70b1-4028-95f1-ec92d35cb51e)?

hm.. on a second look, the docs mention here an unsigned type (https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpefs/1c412a84-0776-4984-b35c-3f0445fcae65). This looks contrary. Some changes here in this MR change `INT32` to `NTSTATUS`, some change `UINT32` to `NTSTATUS`. Do we use now a signed or an unsigned type?

Reply:
@pnowack basically a signed type `NTSTATUS` (`int32_t` typedef)
the spec is contradictory but all status codes in use are referencing `NTSTATUS` values, so the idea is to use this type instead